Pod/Classes/SKYContainer.h

```diff
@@ -226,10 +226,20 @@ typedef void (^SKYContainerUserOperationActionCompletion)(SKYUser *user, NSError
  Registers a device token for push notification.
  */
 - (void)registerRemoteNotificationDeviceToken:(NSData *)deviceToken
-                            completionHandler:(void (^)(NSString *, NSError *))completionHandler;
+                            completionHandler:(void (^)(NSString *, NSError *))completionHandler
+    __deprecated;
 
 /**
- Registers a device without device token.
+ Registers a device token for push notification.
+ When the user is no longer associated to the device, you should call
+ -[unregisterDeviceCompletionHandler].
```

Comment:
I think the better style is to say `-[SKYContainer unregisterDeviceCompletionHandler:]`.